app/views/admin/users/_user_results.html.haml

```diff
@@ -23,4 +25,9 @@
   = paginate @users
 
 - else
-  %p= "No users matched that search"
+  - if params[:search].present?
+    %p No users matched your search.
+  - else
+    %p
+      There are no users,
+      = link_to "click here to create a new user.", new_path_for_user
```

Comment:
I'll add this to the generator code. I'm thinking about doing a blog post something like "Creating the perfect CRUD interface" and getting some of these decisions on a page.